apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.ValueConversion.cs

```diff
@@ -603,14 +604,16 @@ private object ConvertToClrTypeImpl(Value wireValue, System.Type targetClrType,
                         return null;
                     case Value.KindOneofCase.StringValue:
                         return SpannerNumeric.Parse(wireValue.StringValue);
+                    case Value.KindOneofCase.NumberValue:
```

Comment:
Not specific to this line: this method is really getting out of hand now. Separate to this PR, we should try to refactor it.